app/util/conf/base_config_loader.py

```diff
@@ -135,6 +137,21 @@ def load_from_config_file(self, config, config_filename):
         if self.CONFIG_FILE_SECTION:
             self._load_section_from_config_file(config, config_filename, self.CONFIG_FILE_SECTION)
 
+    def _override_cacerts(self):
+        """
+        Override path for requests certs if this is a frozen executable run.
+        SSL cacerts cannot be found when running frozen CR package. Workaround the issue by bundling the cacert.pem
+        file into frozen package and at startup update the environment to point to these cacerts.
+        """
+        if getattr(sys, 'frozen', False):
+            root_directory = dirname(sys.executable)
+            cacert_file = '{}/cacert.pem'.format(root_directory)
+            if isfile(cacert_file):
+                environ['REQUESTS_CA_BUNDLE'] = cacert_file
+            else:
+                print('Missing SSL cacerts file {} in the frozen ClusterRunner package.'.format(cacert_file))
+                sys.exit(1)
```

Comment:
Seems like this should be an exception instead of printing and calling sys.exit. This class should only be responsible for parsing config. Any prints or calls to sys.exit should probably be in main.py -- what do you think?